pkg/operator/ceph/config/livenessprobe.go

```diff
@@ -39,10 +39,21 @@ func ConfigureLivenessProbe(daemon rookv1.KeyType, container v1.Container, healt
 	if _, ok := healthCheck.LivenessProbe[daemon]; ok {
 		if !healthCheck.LivenessProbe[daemon].Disabled {
 			probe := probeFnMap[daemon](healthCheck)
-
 			// If the spec value is empty, let's use a default
 			if probe != nil {
 				container.LivenessProbe = probe
+				// In case of any empty field of probe struct, we are setting default values.
+				if probe.FailureThreshold == 0 {
```

Comment:
Are all these fields actually required? Perhaps we actually only need to override a couple of them. For example, I only see that rook is setting the `Handler` and `InitialDelaySeconds` on all the probes we are creating. The handler may be the only one we need to set here, and the others can be left alone to pick up the K8s defaults. Let's just do the bare minimum so the pod spec won't fail.